Interop/Runtime/src/native/kotlin/kotlinx/cinterop/NativeMem.kt

```diff
@@ -150,15 +150,45 @@ fun CPointer<UShortVar>.toKString(): String {
     while (nativeBytes[length] != 0.toUShort()) {
         ++length
     }
-    val bytes = CharArray(length)
+    val chars = CharArray(length)
     var index = 0
     while (index < length) {
-        bytes[index] = nativeBytes[index].toShort().toChar()
+        chars[index] = nativeBytes[index].toShort().toChar()
         ++index
     }
-    return String(bytes)
+    return String(chars)
 }
 
+fun CPointer<IntVar>.toKString(): String {
```

Comment:
Shouldn't we mention UTF-32 explicitly in the name of this function?

Reply:
How would you suggest to call it?

Reply:
I don't know. Maybe `utf32ToKString`, `toKStringFromUTF32`, `decodeUtf32` or `decodeFromUtf32` etc.
The second variant appears to be more discoverable (considering existing `toKString` functions), but we may decide to cut discoverability down to be able to choose better name.

Reply:
toKStringFromUtf32 looks OKish to me.